dali/sasha/saserver.cpp

```diff
@@ -75,6 +76,7 @@ static void AddServers()
     servers.append(*createSashaDaFSMonitorServer());
     servers.append(*createSashaQMonitorServer());
     servers.append(*createSashaFileExpiryServer()); 
+    servers.append(*createSashaDebugHousekeepingServer());
```

Comment:
this is BM only, and this service will not be useable there, this line should be removed.